contracts/mock/MockUserMintableERC721.sol

```diff
@@ -164,7 +164,10 @@ contract MockUserMintableERC721 is UserMintableERC721 {
     bytes32[] calldata singingRootProof,
     bytes calldata signature,
     bytes32 salt,
-    bytes32[] calldata proof
+    bytes32[] calldata proof,
```

Comment:
The stack to deep error from the tests means that the number of local vars is to big.(Method params and function scoped vars). We could use the array/index pattern in order reduce the number of method params. for ex group all the bytes32 in array in a specific order